job_list_params.go

```diff
@@ -0,0 +1,271 @@
+package river
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/riverqueue/river/internal/dbadapter"
+	"github.com/riverqueue/river/rivertype"
+)
+
+// JobListPaginationCursor is used to specify a starting point for a paginated
+// job list query.
+type JobListPaginationCursor struct {
```

Comment:
Shed color, but IMO `JobListCursor` might be an adequate name for this that's a little less of a mouthful. IMO use of "cursor" implies pagination.

Reply:
Thanks, renamed it ✌️